.changeset/cool-maps-camp.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We should have a changeset to introduce the new packages and why we are adding them